### PR TITLE
[WIP] V2 piet

### PR DESCRIPTION
**Pull request by pietgeursen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1,100 @@
...
-const feathers = require('feathers-client')
-const fetch = require('isomorphic-fetch')
+## API
 
-const client = feathers()
-  .configure(feathers.fetch(fetch))
-
-module.exports = client
-```
+### var reducer = createReducer(opts={})
```
Comment (ahdinosaur) [inline README.md]: maybe show how `createReducer` is attached to `require('feathers-action')`?

File: README.md [hunk 2]
```diff
@@ -1,81 +1,100 @@
...
+Creates a new reducer to pass to `redux.createStore`.
+Valid `opts` keys include:
+- `Resource` (required) - A tcomb named list type describing the resource.
 
+eg:
 ```js
-// types.js
-const t = require('tcomb')
+const feathersAction = require('feathers-action')
+const tc = require('tcomb')
```
Comment (ahdinosaur) [inline README.md]: lowercase `tc` here, while uppercase `Tc` below.

File: README.md [hunk 3]
```diff
@@ -1,81 +1,100 @@
...
-// or actionCreators = createActionCreators(...)
+const client = feathers().configure(...)
 
-module.exports = actions
+const middleware = createMiddleware({ client })
+const enhancer = applyMiddleware(middleware) 
+const store = createStore(state => state, {}, enhancer)
 ```
 
+### var action = createActions(opts={})
```
Comment (ahdinosaur) [inline README.md]: `actions`

**Review by ahdinosaur — APPROVED** [review]
hey this is rad. :+1:

File: README.md (hunk 1 above)
Comment (pietgeursen) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (pietgeursen) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (pietgeursen) [reply README.md]: done

**Pull request closed without merging**